code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -201,20 +201,37 @@
 /obj/item/weapon/melee/energy/axe/attack_self(mob/user as mob)
 	src.active = !( src.active )
 	if (src.active)
-		to_chat(user, "<span class='notice'>The axe is now energised.</span>")
+		to_chat(user, "<span class='notice'>\The [src] is now energised.</span>")
 		src.force = 150
 		src.icon_state = "axe1"
 		src.w_class = W_CLASS_HUGE
 		src.sharpness = 1.5
 	else
-		to_chat(user, "<span class='notice'>The axe can now be concealed.</span>")
+		to_chat(user, "<span class='notice'>\The [src] can now be concealed.</span>")
 		src.force = initial(src.force)
 		src.icon_state = initial(src.icon_state)
 		src.w_class = initial(src.w_class)
 		src.sharpness = initial(src.sharpness)
 	src.add_fingerprint(user)
 	return
-
+	
+/obj/item/weapon/melee/energy/axe/rusty/attack_self(mob/user as mob)
```

Comment:
```
mytype
    value = 10

mytype/proc/myproc()
    do_something(value)

mytype/mychild
    value = 15
```

Now mytype/mychild's myproc() uses 15 in the do_something, instead of 10.